kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -290,6 +290,11 @@ form .input-group {
     background-color: #40775F !important;
 }
 
+.login-btn, btn-block {
+    display: block;
+    width: 100%;
```

Comment:
No - this validation code is either happening in facility/api_resources.py or in distributed/static/js/user/views.js

Incorrect popover behaviour is probably happening in the latter.